alpaca_trade_api/stream.py

```diff
@@ -376,6 +379,7 @@ def __init__(self,
         self._ws = None
         self._running = False
         self._stop_stream_queue = queue.Queue()
+        self._run_forever_flag = False
```

Comment:
What is the purpose of this flag? What functionality does it add over `while True`?